### ISSUE TITLE
[H74x/H75x and G0]: Probing fails only on raspberry pi 3B+

### ISSUE THREAD

**Issue by OSHW-Rico** [stlink-org/stlink#1324]
- [x] Programmer/board type: [V2] (e.g STLINK /V1, /V2, /V2-onboard, /V2-clone, /V3)
- [x] Operating system an version: [Raspian] (e.g Linux, Windows)
- [x] **stlink tools version** and/or git commit hash: [1.7.0] (e.g v1.6.1/git-d0416149)
- [x] stlink commandline tool name: [st-info] (e.g `st-info`, `st-flash`, `st-trace`, `st-util`)
- [x] Target chip (and board, if applicable): [H7 and G0] (e.g STM32F103C8T6 (NUCLEO-F103RB))

When `st-info --probe` from the raspberry pi with Raspian Bullseye:
```

Found 1 stlink programmers
 serial:     543f6906783f565333341167
 hla-serial: "\x54\x3f\x69\x06\x78\x3f\x56\x53\x33\x34\x11\x67"
 flash:      0 (pagesize: 0)
 sram:       0
 chipid:     0x0000
 descr:      unknown device
```
When connecting to an amd64 Laptop Ubuntu:

```
  version:    V2J41S7
  serial:     54FF69067880565333341167
  flash:      2097152 (pagesize: 131072)
  sram:       131072
  chipid:     0x0450
  descr:      H74x/H75x
```
I tried with a V3, same thing. 
Any ideas?

**Comment by Nightwalker-87** [comment]
The output shows that the version of the toolset can not be the same. The lower output is correct.

**Comment by OSHW-Rico** [comment]
What do you mean by toolset?

It's the same thing hardwire wise, took it out from one USB port to the other.

Software wise, it happens with a fresh compiled master branch and the default version from the package manager. (Don't remember the version for that, but I can check if it is important)

**Comment by Nightwalker-87** [comment]
... the stlink tools, this toolset.

**Comment by OSHW-Rico** [comment]
I will check again tomorrow. Should I compile and test with the master branch or the distro version?

Anything else I can collect if the error persists?

**Comment by Nightwalker-87** [comment]
I've just edited the previous reply.

**Comment by OSHW-Rico** [comment]
Okay, thank you.

 I tried development at first but encountered some error. I will try again and report back tomorrow.

**Comment by Nightwalker-87** [comment]
Thanks for the feedback. :+1:  I'll mark this as resolved, but leave the ticket open to have it closed by automation.
In this way we can match the resolving commit or PR to have our changelog updated respectively.

**Comment by Nightwalker-87** [comment]
Fixed in #1052 and thus in Release 1.7.0.
